### PR TITLE
Feat: Jitsi video implemented in screen

### PR DESCRIPTION
**Pull request by AdarPascal**
… guest)

### CODE REVIEW

File: src/common/VideoConference/VideoConference.jsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+            const api = new window.JitsiMeetExternalAPI(domain, options);
+            api.addEventListener('videoConferenceJoined', () => {
+                setLoading(false);
+                api.executeCommand('displayName', displayName);
+            });
+            api.addEventListener('readyToClose', () => {
+                setDisplayVideoConference(false);
+            });
+        // eslint-disable-next-line no-empty
+        } catch (err) {}
```
Comment (carmeltsadok) [inline src/common/VideoConference/VideoConference.jsx:53]: toast an error

File: src/common/VideoConference/VideoConference.jsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        display: 'flex',
+        justifyContent: 'center',
+        marginTop: '10rem',
+    };
+
+    const jitsiContainerStyle = {
+        display: (loading ? 'none' : 'block'),
+        width: '100%',
+        height: '100%',
+    };
```
Comment (carmeltsadok) [inline src/common/VideoConference/VideoConference.jsx:21]: add to styles file and use `className` property on the divs themselves

File: src/pages/InterviewDashboard/components/Malshab/Malshab.jsx [hunk 3]
```diff
@@ -1,17 +1,60 @@
...
+        onChange={(e) => setDisplayName(e.target.value)}
+      />
+    </div>
+    <div className={classes.btnContainer}>
+      <Button variant='contained' color='primary' size='large' onClick={() => setDisplayVideoConference(true)}>
+        {buttonText}
+      </Button>
+    </div>
+  </div>
+);
```
Comment (carmeltsadok) [inline src/pages/InterviewDashboard/components/Malshab/Malshab.jsx:30]: move to a new component

**Review by carmeltsadok — CHANGES_REQUESTED** [review]

File: src/common/VideoConference/VideoConference.styles.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(() => ({
+  containerStyle: {
+    // width: '800px',
+    // height: '400px',
+  },
```
Comment (carmeltsadok) [inline src/common/VideoConference/VideoConference.styles.js:7]: remove style and remove it from div

File: src/common/VideoConference/VideoConference.jsx [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+  };
+
+  useEffect(() => {
+    if (window.JitsiMeetExternalAPI && !document.getElementById('largeVideo')) startConference();
+  }, []);
+
+  return (
+    <div className={classes.input}>
+      {loading && (
+        <div className={classes.containerStyle}>
```
Comment (carmeltsadok) [inline src/common/VideoConference/VideoConference.jsx:53]: isn't it a useless div?

File: src/pages/InterviewDashboard/DisplayNamePick/DisplayNamePick.jsx [hunk 6]
```diff
@@ -0,0 +1,17 @@
+import { Button } from '@material-ui/core';
+import TextField from '@material-ui/core/TextField';
+
+const DisplayNamePick = ({ displayName, setDisplayName, setDisplayVideoConference, classes, header, placeholder, buttonText }) => (
+  <div className={classes.displayNamePickContainer}>
+    <div style={{ display: 'grid', justifyContent: 'center' }}>
```
Comment (carmeltsadok) [inline src/pages/InterviewDashboard/DisplayNamePick/DisplayNamePick.jsx:6]: add a classname

**Review by carmeltsadok — CHANGES_REQUESTED** [review]

**Review by yubbi99 — APPROVED** [review]

**Review by carmeltsadok — APPROVED** [review]

**Pull request merged**